templates/CRM/Admin/Form/Preferences/Display.tpl

```diff
@@ -155,11 +155,11 @@
 
     <tr class="crm-preferences-display-form-block-preserve_activity_tab_filter">
       <td class="label"></td>
-      <td>{$form.preserve_activity_tab_filter.html} {$form.preserve_activity_tab_filter.label}</td>
+      <td>{$form.preserve_activity_tab_filter.html} </td>
     </tr>
     <tr class="crm-preferences-display-form-block-description">
       <td>&nbsp;</td>
-      <td class="description">{ts}When enabled, any filter settings a user selects on the contact's Activity tab will be remembered as they visit other contacts.{/ts}
+      <td class="description">{$settings_fields.preserve_activity_tab_filter.description}
```

Comment:
I think the </td> should be on the line above.

Reply:
fixed